testdata/firebase_config_bad_key.json

```diff
@@ -0,0 +1,3 @@
+{
+  "databaseUrl": "https://hipster-chat.firebaseio.mock"
```

Comment:
Do we really care that it fails with a case mismatch? I think json.Unmarshal usually handles case mismatches. So maybe we should actually let this case work, or at least ignore it, and test a bad key that isn't just a case difference from a good key?

Reply:
You are right, json ignores case.  (I was thinking of catching a subtle error..)
Should I change the underlying code to ignore case as well, or just have the test case explicitly fail? 
I think I'll just change the test case, and let the TODO (disallowed) take care of that..